e2e_test/e2e_test.go

```diff
@@ -36,3 +37,44 @@ func TestSendCelo(t *testing.T) {
 	err = network.AwaitTransactions(ctx, tx)
 	require.NoError(t, err)
 }
+
+func TestSingleNodeManyTxs(t *testing.T) {
+	rounds := 100
+	accounts := test.Accounts(1)
+	gc := test.GenesisConfig(accounts)
+	gc.Istanbul.Epoch = uint64(rounds) * 5 // avoid the epoch for this test
+	network, err := test.NewNetwork(accounts, gc)
+	require.NoError(t, err)
+	defer network.Shutdown()
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	for r := 0; r < rounds; r++ {
+		tx, err := network[0].SendCelo(ctx, common.Address{}, 1)
+		require.NoError(t, err)
+		require.NotNil(t, tx)
+		err = network.AwaitTransactions(ctx, tx)
+		require.NoError(t, err)
+	}
```

Comment:
Calling `AwaitTransactions` inside the loop means that we wait for each tx in turn so each block will have 1 tx and so we wait for at least 100 blocks. If we instead just wait at the end then we will likely end up processing more than 1 tx per block and the test should be significantly faster since we are not constantly waiting.

```suggestion
	txs := make([]*types.Transaction, rounds)
	for r := 0; r < rounds; r++ {
		tx, err := network[0].SendCelo(ctx, common.Address{}, 1)
		require.NoError(t, err)
		require.NotNil(t, tx)
		txs[r] = tx
	}

	err = network.AwaitTransactions(ctx, txs...)
	require.NoError(t, err)

```

Reply:
This was actually intentional. If we want more Txs then I can fire up more txs per round